modules/log6x-logging-loki-gui-install.adoc

```diff
@@ -0,0 +1,125 @@
+// Module is included in the following assemblies:
+//
+// * observability/logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="6x-logging-loki-gui-install_{context}"]
+= Installing {logging-uc} and the {loki-op} using the web console
+
+To install and configure logging on your {product-title} cluster, an Operator such as {loki-op} for log storage must be installed first. This can be done from the OperatorHub within the web console.
+
+.Prerequisites
+
+* You have access to a supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, {rh-storage}).
+* You have administrator permissions.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. In the {product-title} web console *Administrator* perspective, go to *Operators* -> *OperatorHub*.
+
+. Type {loki-op} in the *Filter by keyword* field. Click *{loki-op}* in the list of available Operators, and then click *Install*.
++
+[IMPORTANT]
+====
+The Community {loki-op} is not supported by Red{nbsp}Hat.
+====
+
+. Select *stable* or *stable-x.y* as the *Update channel*.
++
+--
+include::snippets/logging-stable-updates-snip.adoc[]
+--
++
+The {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, so the *Installation mode* and *Installed Namespace* are already selected. If this namespace does not already exist, it is created for you.
```

Comment:
```suggestion
The {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, so the *Installation mode* and *Installed Namespace* are already selected. If this namespace does not already exist, it will be created for you.
```

Reply:
done